nullaway/src/main/java/com/uber/nullaway/dataflow/MapKey.java

```diff
@@ -0,0 +1,4 @@
+package com.uber.nullaway.dataflow;
+
+/** Marker interface to implement a union type of classes that can be used as AccessPath.mapKey */
+interface MapKey {}
```

Comment:
This needs to be outside `AccessPath.java` now to avoid circular definition issues.